models/human.rb

```diff
@@ -25,5 +25,218 @@ def job_title=(value)
   def job_title
     @job_title
   end
+
+attr_accessor :age
+
+  # def age=(value)
+  #   @age = value
+  # end
+  #
+  # def age
+  #   @age
+  # end
+
+  def can_drink? (country = "Australia")
+    @age >= AGES[country]
+  end
 end
 
+
+AGES = {
```

Comment:
great foresight here...  Only small suggestion, this would be better to put in another file (or in Rails we would of had it in a DB table) to reduce the noise in the model
In this case it might be better to make this another class that you can ask for drinking_ages (as per comment below in specs). We can discuss this further next week